### PR TITLE
Update monitoring docs and add dashboard links

### PR DESCRIPTION
**Pull request by kmandrika**
Description here... 

Checklist (if applicable):
- [ ] Tested (manually, unit tested, etc.)
- [ ] Changelog updated
- [x] Docs updated

### CODE REVIEW

File: docs/plugins/google-cloud.md [hunk 1]
```diff
@@ -152,4 +152,4 @@ This option will force Genkit to export telemetry and log data when running in t
 
 ## Test your integration
 
-When configuring the plugin, use `forceDevExport: true` to enable telemetry export for local runs. Navigate to the Google Cloud Logs, Metrics, or Trace Explorer to view telemetry. Alternatively, navigate to the Firebase AI Monitoring dashboard for an AI-idiomatic view of telemetry.
+When configuring the plugin, use `forceDevExport: true` to enable telemetry export for local runs. Navigate to the Google Cloud Logs, Metrics, or Trace Explorer to view telemetry. Alternatively, navigate to the [Firebase AI Monitoring dashboard](https://console.firebase.google.com/project/_/genai_monitoring) for an AI-idiomatic view of telemetry.
```
Comment (schnecle) [inline docs/plugins/google-cloud.md:155]: Customers still won't be able to just see it in the console until we move out of private preview - want to link to the form to sign up?

File: docs/monitoring.md [hunk 2]
```diff
@@ -2,39 +2,4 @@
...
-
-```ts
-import { firebase } from '@genkit-ai/plugin-firebase';
-
-configureGenkit({
-  plugins: [firebase()],
-  traceStore: 'firebase',
-});
-```
+telemetry data. The [Google Cloud plugin](./plugins/google-cloud.md) and the [Firebase plugin](./plugins/firebase.md) both export telemetry to Cloud's operations suite. Uisng either plugin poweres the [Firebase AI Monitoring dashboard (private preview)](https://forms.gle/Lp5S1NxbZUXsWc457) that has an AI-idiomatic view of telemetry data.
```
Comment (schnecle) [inline docs/monitoring.md:5]: nit: Uising -> Using

**Review by schnecle — APPROVED** [review]
LGTM! Do we need to get a Kevin or Chris review on this?

File: docs/monitoring.md [hunk 3]
```diff
@@ -1,40 +1,57 @@
...
-  traceStore: 'firebase',
-});
+// Set the desired log level
+logger.setLogLevel('debug');
+```
+
+## Trace Storage and Developer UI
+Traces are automatically captured and can be viewed in the Genkit Developer UI. To start the UI:
+```
+genkit ui:start
```
Comment (schnecle) [inline docs/monitoring.md:55]: Should this now be the `genkit start -- <watch your code>` command?

File: docs/plugins/google-cloud.md [hunk 4]
```diff
@@ -152,4 +152,4 @@ This option will force Genkit to export telemetry and log data when running in t
 
 ## Test your integration
 
-When configuring the plugin, use `forceDevExport: true` to enable telemetry export for local runs. Navigate to the Google Cloud Logs, Metrics, or Trace Explorer to view telemetry. Alternatively, navigate to the Firebase AI Monitoring dashboard for an AI-idiomatic view of telemetry.
+When configuring the plugin, use `forceDevExport: true` to enable telemetry export for local runs. Navigate to the Google Cloud Logs, Metrics, or Trace Explorer to view telemetry. Alternatively, navigate to the [Firebase AI Monitoring dashboard (private preview)](https://forms.gle/Lp5S1NxbZUXsWc457) for an AI-idiomatic view of telemetry.
```
Comment (schnecle) [inline docs/plugins/google-cloud.md:155]: Do we want to show them where this configuration goes? This could be a touch confusing.

**Review by schnecle — COMMENTED** [review]
A couple small things, but this looks great!

File: docs/plugins/google-cloud.md (hunk 4 above)
Comment (kmandrika) [reply docs/plugins/google-cloud.md:155]: There is an example of this above under Plugin Configuration, so I am tempted to say no.

File: docs/monitoring.md [hunk 5]
```diff
@@ -1,40 +1,57 @@
...
-The `traceStore` option is complementary to the telemetry instrumentation. It
-lets you inspect your traces for your flow runs in the Genkit Developer UI. It
-requires a separate configuration which provides a trace storage implementation.
-The `firebase` plugin offers a Firestore-based implementation. This
-configuration is optional, but is recommended because it lets you inspect and
-debug issues in production. When using Firestore-based trace storage you will
-want to enable TTL for the trace documents:
-https://firebase.google.com/docs/firestore/ttl
+## Logging
+Genkit provides a centralized logging system that can be configured using the logging module.
```
Comment (alexsinger) [inline docs/monitoring.md:43]: How do these logs relate to the telemetry configuration above? Should we add clarity?

**Review by kevinthecheung — APPROVED** [review]

File: docs/monitoring.md (hunk 3 above)
Comment (chrisraygill) [reply docs/monitoring.md:55]: That's correct. I think linking to `/docs/genkit/devtools` is probably better than re-explaining the whole thing.

**Pull request merged**